ztp/gitops-subscriptions/argocd/resource-hook-example/policygentemplates/common-ranGen.yaml

```diff
@@ -55,28 +51,16 @@ spec:
     - fileName: StorageSubscription.yaml
       policyName: "storage-sub-policy"
     # These CRs are in support of installation from a disconnected registry
-    - fileName: ClusterLogCatSource.yaml
-      policyName: "log-sub-policy"
-      spec:
-        image: registry.example.com:5000/olm/far-edge-clo:latest
-    - fileName: PtpCatSource.yaml
-      policyName: "ptp-sub-policy"
-      spec:
-        image: registry.example.com:5000/olm/far-edge-ptp:latest
-    - fileName: StorageCatSource.yaml
-      policyName: "storage-sub-policy"
-      spec:
-        image: registry.example.com:5000/olm/far-edge-lso:latest
-    - fileName: SriovCatSource.yaml
-      policyName: "sriov-sub-policy"
+    - fileName: DefaultCatsrc.yaml
```

Comment:
what's this ?

Reply:
This is the "single" catalogsource CR. All other catalogsources are disabled (including the default catalogsources which is done by the OperatorHub CR).